wordpress/tests/test_wpuser.py

```diff
@@ -0,0 +1,79 @@
+import responses
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+from wordpress.user import WPUser
+from wordpress.client import APIClient
+
+from api.tasks import update_user_as_partner
+
+
+class WPUserTest(TestCase):
+    def setUp(self):
+        self.mock_client = MagicMock()
+        self.api_client = APIClient()
+        self.email = "email@email.com"
+        self.user_id = 13
+        self.partner_role = "test_partner_role"
+
+    @responses.activate
+    def test_fetch_wp_user(self):
+        wp_user = WPUser(self.api_client)
+        responses.add(
+            responses.GET,
+            f"https://test_wp_host/wp-json/wp/v2/users/?search={self.email}",
+            status=200,
+            json=[{"id": self.user_id}],
+        )
+
+        responses.add(
+            responses.GET,
+            f"https://test_wp_host/wp-json/wp/v2/users/{self.user_id}?context=edit",
+            status=200,
+            json={
+                "id": self.user_id,
+                "username": "testusername",
+                "email": self.email,
+                "roles": ["testrole"],
+            },
+        )
```

Comment:
I don't mean to change this now but we have a bit of a mess with response stubs. See https://github.com/coopdevs/lazona_connector/issues/13. But I was also trying to get the latest changes on https://github.com/coopdevs/lazona_connector/pull/64. We have to stick to one.